app-modules/inventory-management/src/Filament/Resources/AssetResource/Pages/ViewAsset.php

```diff
@@ -54,21 +53,18 @@ public function infolist(Infolist $infolist): Infolist
     {
         return $infolist
             ->schema([
-                Section::make()
```

Comment:
We don't want to remove the `Section` here as that provides the nice box section around the content.

We just need to make this section have two columns.

Reply:
Done